sdcm/cluster.py

```diff
@@ -1319,6 +1320,19 @@ def wait_db_up(self, verbose=True, timeout=3600):
         except Exception as details:  # pylint: disable=broad-except
             self.log.error('Failed to report housekeeping uuid. Error details: %s', details)
 
+    def manager_agent_up(self):
+        if self.is_port_used(port=self.MANAGER_AGENT_PORT, service_name="scylla-manager-agent"):
+            # When the agent is IP, it should answer an http request of https://NODE_IP:10001/ping with status code 204
+            response = requests.get(f"https://{self.ip_address}:10001/ping", verify=False)
+            return response.status_code == 204
+        return False
+
+    def wait_agent_up(self, verbose=True, timeout=180):
```

Comment:
`wait_agent_up` -> `wait_manager_agent_up`

Reply:
Done and done